src/org/exist/storage/NativeBroker.java

```diff
@@ -1205,6 +1213,10 @@ private void copyModeAndAcl(final Permission srcPermission, final Permission des
         }
     }
 
+    private boolean isSubCollection(final Collection col, final Collection sub) {
+        return sub.getURI().startsWith(col.getURI());
```

Comment:
Every collection is a sub-collection of `/db`, so how does this check cope with that?